### PR TITLE
Acrn

### PR DESCRIPTION
**Pull request by DunderRoffe**
This PR adds ACRN functionality to the ARP when acrn-layer is present in bblayers.

### CODE REVIEW

**Comment by DunderRoffe** [comment]
Could you please review commit and file structure. It is verified to work on actual target so I am mostly looking for reviews on semantics etc

File: layers/b2qt/recipes-bsp/lsr/lsfsync_git.bb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Copyright (C) 2019 Viktor Sjölind <vsjolind@luxoft.com>
```
Comment (sashko) [inline layers/b2qt/recipes-bsp/lsr/lsfsync_git.bb]: Luxoft Sweden

File: layers/b2qt/recipes-bsp/lsr/lsfsync_git.bb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+# Released under the MIT license (see COPYING.MIT for the terms)
```
Comment (sashko) [inline layers/b2qt/recipes-bsp/lsr/lsfsync_git.bb]: License identifier instead

File: layers/b2qt/recipes-bsp/lsr/lsfsync/lsfsync.service [hunk 3]
```diff
@@ -0,0 +1,14 @@
+[Unit]
```
Comment (martin-ejdestig) [inline layers/b2qt/recipes-bsp/lsr/lsfsync/lsfsync.service:1]: Move the service file to the lsfsync repo?

Comment (DunderRoffe) [reply layers/b2qt/recipes-bsp/lsr/lsfsync/lsfsync.service:1]: No. IMHO .service files should be provided when packaging which means in the yocto-layer in this case. The reason I believe so is that where the binary that the service file refers to is placed is decided at the packaging stage. Meaning, in the repo one does not know where the binary will be locate and should not enforce the placement upon the packaging party.

Comment (martin-ejdestig) [reply layers/b2qt/recipes-bsp/lsr/lsfsync/lsfsync.service:1]: > IMHO .service files should be provided when packaging 

Definitely not.

Normally the service file is generated from a template that has e.g. @bindir@ and other template variables in it if needed.

But just having it here is perhaps easier in this case.

(But also... doing it this way is a reminder/remnant of the past dark (sysv init) days when distro packagers had to write their own init scripts. ;)

Comment (DunderRoffe) [reply layers/b2qt/recipes-bsp/lsr/lsfsync/lsfsync.service:1]: Hmm maybe you have a point. Never seen/thought of having .in files for service files but indeed that is how big projects seem to do it.
As painful as it is maybe I should try to learn from this :)
Also either your smiley  is incomplete or you are missing a closing parenthesis :angry:

Comment (DunderRoffe) [reply layers/b2qt/recipes-bsp/lsr/lsfsync/lsfsync.service:1]: I have decided that this is future @DunderRoffe s problem. So lets fix it outside of this PR. Adding a build system for lsfsync is too much to do before merging this.
Are you OK with doing it later ? @martin-ejdestig

Comment (martin-ejdestig) [reply layers/b2qt/recipes-bsp/lsr/lsfsync/lsfsync.service:1]: "It's not just good, it's good enough." (Have not requested changes. :)

**Pull request merged**